backend/src/routes/user.ts

```diff
@@ -563,9 +563,19 @@ router.get(
           getPastSessionPromise,
           getPairedMentorIdPromise,
         ]);
+        const mentor = await Mentor.findById(pairedMentorId);
+        if (!mentor) {
+          throw ServiceError.MENTOR_WAS_NOT_FOUND;
+        }
         res.status(200).send({
           status: mentee.status,
-          sessionId: upcomingSessionId ?? pastSessionId,
+          name: mentee.name,
+          mentorName: mentor?.name ?? "N/A",
+          mentorMajor: mentor?.major ?? "N/A",
+          mentorUniversity: mentor?.college ?? "N/A",
+          mentorCareer: mentor?.career ?? "N/A",
+          upcomingSessionId: upcomingSessionId ?? "N/A",
+          pastSessionId: pastSessionId ?? "N/A",
```

Comment:
I think it's good that we're returning both the upcoming session id and past session id. I was just wondering if in the session document we store some field to state whether a session was canceled or rescheduled. I only ask because on Frontend we have to update the session card based on whether a session is scheduled, canceled, or rescheduled for the mentee home page. If you think that we can implement that feature on the Home Page with the current route as it stands then I think its good to merge. Otherwise, it might be useful to look into changing the sessions route to reflect whether a session was canceled, rescheduled, or no changes.